### PR TITLE
Fix: Improved button design

### PR DESCRIPTION
**Pull request by Kireck211**
## Summary
Fixes the button design, because it was a little odd to be changed the width depending on the content of the animal card.

## Manual test
1. Pull down the changes
2. Run the command: `npm run start`
3. Verify that the button stays the same width regardless of the content of the card

## Ticket related
[Trello Ticket Fix buttons size](https://trello.com/c/gpQu06uh/13-front-fix-buttons-size-1)

### CODE REVIEW

File: public/stylesheets/style.less [hunk 1]
```diff
@@ -42,14 +42,15 @@ div.card {
     display: flex;
     flex-direction: column;
     justify-content: center;
+    width: 100%;
```
Comment (erickwize) [inline public/stylesheets/style.less:45]: Why did you added this code?

File: public/stylesheets/style.less [hunk 2]
```diff
@@ -42,14 +42,15 @@ div.card {
...
 
     h2 {
       margin: 0;
       padding-bottom: 10px;
     }
 
     a {
-      width: 100%;
```
Comment (erickwize) [inline public/stylesheets/style.less:52]: Is it necessary to remove the width?

**Review by erickwize — CHANGES_REQUESTED** [review]
I leave some comments on the PR

File: public/stylesheets/style.less (hunk 1 above)
Comment (Kireck211) [reply public/stylesheets/style.less:45]: To make the container content take all the remaining space

File: public/stylesheets/style.less (hunk 2 above)
Comment (Kireck211) [reply public/stylesheets/style.less:52]: Yes it is necessary because the user could click on a whitespace and would trigger the change on the page

**Review by erickwize — APPROVED** [review]
LGTM! 👍

**Pull request merged**